codalab/apps/web/static/js/worksheet/table_item_interface.jsx

```diff
@@ -112,6 +113,20 @@ var TableRow = React.createClass({
         this.props.updateRowIndex(this.props.rowIndex);
     },
 
+    handleOptionButtonClick(event) {
```

Comment:
Change the variable name `event` to something else, like `event_`, because in Internet Explorer `event` is a global variable.